addons/laser/stringtable.xml

```diff
@@ -51,22 +51,56 @@
             <Chinese>雷射碼</Chinese>
             <Turkish>Lazer Kod</Turkish>
         </Key>
-        <Key ID="STR_ACE_Laser_laserCodeUp">
-            <English>Laser - Cycle Code Up</English>
-            <German>Lasercode +</German>
-            <Polish>Laser - Następny kod</Polish>
-            <French>Code laser +</French>
-            <Russian>Лазер - увеличить частоту</Russian>
-            <Portuguese>Laser - Alternar Código para Cima</Portuguese>
-            <Hungarian>Lézer - kódciklus növelése</Hungarian>
-            <Spanish>Láser - Aumentar código</Spanish>
-            <Czech>Laser - Kód +</Czech>
-            <Italian>Laser - Cambia codice +</Italian>
-            <Japanese>レーザ - コードの数値を増やす</Japanese>
-            <Korean>레이저 - 코드 순환 위</Korean>
-            <Chinesesimp>激光—循环切换激光码 上</Chinesesimp>
-            <Chinese>雷射 - 循環切換雷射碼 上</Chinese>
-            <Turkish>Lazer - Çevrim Kodu Yukarı</Turkish>
+        <Key ID="STR_ACE_Laser_laserCodeUpOnes">
+            <English>Laser - Cycle Code Up - XXX#</English>
+            <German>Lasercode +XXX#</German>
```

Comment:
not sure why german/french are so different
should this be "groBcoden" or something?

Reply:
I honestly don't know, and I wanted to mess with existing translations as little as possible in this case if it could at all be avoided.